### DISCUSSION TITLE
MATLAB Tutorial Exercise Solutions

### DISCUSSION THREAD

**Discussion by chrischambers888** [chulminy/CIVE497-CIVE700#144] [Q&A] [answered]
Hi, I have completed the intro tutorials for MATLAB and would like to compare my solutions with the ideal solutions.  Are the solutions posted somewhere?  If not, could you please post them?

**Comment by chulminy** [comment, marked as answer]
Great questions. I can show the model solution tomorrow. 

Chul Min
